test/fixtures/rules/std_rule_cases/LT02-indent.yml

```diff
@@ -1815,6 +1815,84 @@ test_fix_untaken_positive_4433:
     )
     ;
 
+test_ansi_group_by_implicit_4539:
+  # https://github.com/sqlfluff/sqlfluff/issues/4539
+  pass_str: |
+    SELECT t.col1, t.col2
+    FROM t
+    GROUP BY t.col1, t.col2
+  configs:
+    core:
+      dialect: ansi
+    indentation:
+      allow_implicit_indents: true
+
```

Comment:
I'm a little confused about these test cases - they would still pass even _without_ implicit indents. 🤔 

To test the changes that you're making don't you need a line break before `t.col2` in the `GROUP BY` clause?

Reply:
Was thinking the same thing 🙂